rocketpy/rocket/aero_surface/generic_surface.py

```diff
@@ -85,12 +84,55 @@ def __init__(
         self.cpz = center_of_pressure[2]
         self.name = name
 
-        self.cL = self._process_input(cL, "cL")
-        self.cD = self._process_input(cD, "cD")
-        self.cQ = self._process_input(cQ, "cQ")
-        self.cm = self._process_input(cm, "cm")
-        self.cn = self._process_input(cn, "cn")
-        self.cl = self._process_input(cl, "cl")
+        default_coefficients = self._get_default_coefficients()
+        coefficients = self._complete_coefficients(coefficients, default_coefficients)
+        for coeff, coeff_value in coefficients.items():
+            value = self._process_input(coeff_value, coeff)
+            setattr(self, coeff, value)
+
+    def _get_default_coefficients(self):
+        """Returns default coefficients
+
+        Returns
+        -------
+        default_coefficients: dict
+            Dictionary whose keys are the coefficients names and keys
+            are the default values.
+        """
+        default_coefficients = {
+            "cL": 0,
+            "cQ": 0,
+            "cD": 0,
+            "cm": 0,
+            "cn": 0,
+            "cl": 0,
+        }
+        return default_coefficients
+
+    def _complete_coefficients(self, input_coefficients, default_coefficients):
+        """Completes coefficients dictionary from user input in __init__
+
+        Parameters
+        ----------
+        input_coefficients : str, dict
+            Coefficients dictionary passed by the user. If the user only specifies some
+            of the coefficients, the
+
+        Returns
+        -------
+        coefficients: dict
+            Coefficients dictionary used to setup coefficients attributes
+        """
+
+        if input_coefficients == "all_null":
+            coefficients = default_coefficients
+        else:  # complete user dictionary with null values for the coefficients
+            coefficients = input_coefficients
+            for coeff, value in default_coefficients.items():
+                if coeff not in coefficients.keys():
+                    coefficients[coeff] = value
+
```

Comment:
I think this loop might be a little deficient.

What happens if the dict has a key that is not one of the coefficients names?

Also, since you are looping through the names, you should raise an error in case any keys in the user dict does not have an accepted string as key

Reply:
Added a `_check_coefficients` method to raise an error in such cases.